cms/templates/webinar_page.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% load render_bundle wagtailcore_tags %}
+{% load static wagtailimages_tags wagtailmetadata_tags render_bundle image_version_url wagtailcore_tags tz %}
```

Comment:
Loaded render_bundle and wagtailcore_tags twice. Also, please verify if there are any unused tags.